3rdparty/nuxeo/nuxeo-platform-elasticsearch/src/main/java/org/collectionspace/services/nuxeo/elasticsearch/DefaultESDocumentWriter.java

```diff
@@ -306,7 +307,11 @@ private void denormConceptFields(final DocumentModel doc, final ObjectNode denor
 			for (String content  : contentList) {
 				if (content != null) {
 					final ObjectNode node = objectMapper.createObjectNode();
-					node.put("subject", RefNameUtils.getDisplayName(content));
+					if ("contentPlaces".equals(field)) {
+						node.put("subject", content); // Directly use the value for contentPlaces
+					} else {
+						node.put("subject", RefNameUtils.getDisplayName(content)); // Use RefNameUtils for other fields
+					}
```

Comment:
It looks like [some profiles](https://github.com/collectionspace/cspace-ui-plugin-profile-fcart.js/blob/462b63b14c1745e1718331b098059098c75fb2ef/src/plugins/recordTypes/collectionobject/fields.js#L66) override `contentPlace` to be a RefName, so maybe we should check with `RefNameUtils.isTermRefname` here instead. Makes me wonder if any of the other fields have been overridden as well and have been failing to parse.